src/olmo_core/nn/functional/cross_entropy_loss.py

```diff
@@ -53,11 +53,11 @@ def cross_entropy_loss(
 _fused_cross_entropy_loss: Optional[Callable] = None
 
 try:
-    import olmo_core.triton.cross_entropy_loss as triton_ce_loss
+    # import olmo_core.triton.cross_entropy_loss as triton_ce_loss
+    #_fused_cross_entropy_loss = triton_ce_loss.cross_entropy_loss
 
-    #  import flash_attn.ops.triton.cross_entropy as flash_attn_ce  # type: ignore
-
-    _fused_cross_entropy_loss = triton_ce_loss.cross_entropy_loss
+    import flash_attn.ops.triton.cross_entropy as flash_attn_ce  # type: ignore
```

Comment:
Our in-house triton CE loss was copied directly from the flash-attn repo, so I don't see the point of this.

Reply:
Ok, I took this back out.

Reply:
Do I want compiling and fused loss at the same time?